modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -122,8 +122,17 @@ private void _MenuOptionPressed(long id)
             switch ((MenuOptions)id)
             {
                 case MenuOptions.CreateSln:
-                    CreateProjectSolution();
+                {
+                    if (File.Exists(GodotSharpDirs.ProjectSlnPath))
```

Comment:
```suggestion
                    if (File.Exists(GodotSharpDirs.ProjectSlnPath) || File.Exists(GodotSharpDirs.ProjectCsProjPath))
```
This also overwrites the project file,